lib/cisco_node_utils/bgp.rb

```diff
@@ -94,6 +95,23 @@ def destroy
       router_bgp('no')
     end
 
+    def process_initialized?
+      config_get('bgp', 'process_initialized')
+    end
+
+    def wait_for_process_initialized
+      return unless node.product_id[/N(5|6)/]
+
+      # Hack for slow-start platforms which may have setter failures if the
+      # bgp instance is still initializing when they process. To see this
+      # problem in a sandbox do 'router bgp 1 ; router bgp 1 ; shutdown'.
+      4.times do
+        break if process_initialized?
```

Comment:
I believe you mentioned that the error message is not very meaningful if the process is not initialized?  If that is indeed the case, I would like to see a meaningful message raised in the event that the process does not initialize after 4 attempts.